### PR TITLE
Changed backgrounds and skills

### PR DESCRIPTION
**Pull request by noah-sennett**
Here I've replaced backgrounds.json and skills.json with the equivalent files from noah-sennett/swn-character-creator . The former contains info for all backgrounds (albeit formatted differently from what is currently in the repository). The latter contains additional "implict" information, namely whether the skills should be considered as combat, non-combat, or psychic, which is relevant for certain foci and learning table selections.

### CODE REVIEW

File: weapons.json [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+    "cost":"500",
+    "magazine":"1",
+    "attribute":"Dex",
+    "encumberance":"2",
+    "bonus":"0",
+    "TL":"4",
+    "burst":false,
+    "description":"Bows are uncommon weapons in the far future, though some lostworlder barbarians have nothing better. Conversion bows use special materials to convert the kinetic energy of the draw into a force field “glazing” around the arrow, improving penetration. Bows can be reloaded with a Move action, or faster if the Gunslinger focus is applied."
+  },
+  "gernade":{
```
Comment (tannerstephens) [inline weapons.json:44]: Nit: Grenade

Comment (tannerstephens) [inline weapons.json:44]: Nit: Grenade

Comment (tannerstephens) [inline weapons.json]: Nit: Grenade

**Review by tannerstephens — CHANGES_REQUESTED** [review]

**Comment by noah-sennett** [comment]
Fixed "gernade" typo

**Review by tannerstephens — APPROVED** [review]

**Pull request merged**